pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.md

```diff
@@ -93,13 +89,14 @@ AMP provides a set of layouts that specify how an AMP component behaves in the d
 
 [sourcecode:html]
 <amp-img
-  src="/img/amp.jpg"
-  width="1080"
-  height="610"
-  layout="responsive"
-  alt="an image"
-></amp-img>
-[/sourcecode]
+src="/img/amp.jpg"
+width="1080"
+height="610"
+layout="responsive"
+alt="an image"
+
+> </amp-img>
+> [/sourcecode]
```

Comment:
As I mentioned to @matthiasrohmer, I believe the reason for this strangeness is due to Prettier's Markdown parser to being able to understand how to interpret the `[sourcecode:html]` syntax (Jinja filters) in the middle of the Markdown.

Technically this makes this a new flavor of Markdown, so we may want to teach the parser about AMP Flavored Markdown of we are to expect this to not get strangely formatted like this. We may be able to get away w/ surrounding these with HTML comments too.

Thoughts?

Reply:
I seem to have been mistaken, these are not Jinja macros, but something else. These blocks don't appear in the CommonMark specification either. https://spec.commonmark.org/0.29/